src/app/page.tsx

```diff
@@ -9,96 +9,113 @@ import "primeflex/primeflex.css";
-          min={10}
-          max={40}
-        />
+      <div id="head">
+        <h1>Veamos tu potencial</h1>
+        <p>
+          Aquí una muestra de cómo el tiempo y el interés compuesto pueden
+          ayudarte a hacer crecer tu dinero.
+        </p>
       </div>
-      <Button label="Calcular mis rendimientos" unstyled />
+      <section className="Calc">
+        <form>
+          <div>
+            <label htmlFor="initDepo">Depósito Inicial</label>
+            <InputNumber
+              className="border-circle-top border-circle-bottom"
+              name="initDepo"
+              inputId="currency-us"
+              value={init}
+              onValueChange={(e) => setInit(e.value)}
+              mode="currency"
+              currency="USD"
+              locale="es"
+            />
+          </div>
+          <div className="">
+            <label htmlFor="contr">Aportes</label>
+            <InputNumber
+              className="keyIn"
+              name="contr"
+              inputId="currency-us"
+              value={budget}
+              onValueChange={(e) => setBudget(e.value)}
+              mode="currency"
+              currency="USD"
+              locale="es"
+            />
+            <RadioButton
+              inputId="years"
+              name="contrib"
+              value="Anual"
+              onChange={(e) => setFrequency(e.value)}
+              checked={frequency === "Anual"}
+            />
+            <label htmlFor="years">Anual</label>
+            <RadioButton
+              inputId="months"
+              name="contrib"
+              value="Mensual"
+              onChange={(e) => setFrequency(e.value)}
+              checked={frequency === "Mensual"}
+            />
+            <label htmlFor="months">Mensual</label>
+            <RadioButton
+              inputId="weeks"
+              name="contrib"
+              value="Semanal"
+              onChange={(e) => setFrequency(e.value)}
+              checked={frequency === "Semanal"}
+            />
+            <label htmlFor="weeks">Semanal</label>
+            <RadioButton
+              inputId="days"
+              name="contrib"
+              value="Diario"
+              onChange={(e) => setFrequency(e.value)}
+              checked={frequency === "Diario"}
+            />
+            <label htmlFor="days">Diario</label>
+          </div>
+          <div className="">
+            <label htmlFor="growTime">Plazo de Inversión</label>
+            <p>{`${term} ${isPlural}`}</p>
+            <Slider
+              name="growTime"
+              value={term}
+              onChange={(e) => setTerm(e.value)}
+              className="p-slider p-slider-handle w-20rem"
+              min={1}
+              max={40}
+            />
+          </div>
+          <div>
+            <label htmlFor="percnt">Retorno anual promedio</label>
+            <InputNumber
+              className="keyIn"
+              name="percnt"
```

Comment:
why not the full word for the name?

```suggestion
              name="percent"
```

Reply:
Better! Used to abbreviate virtually everything. Bad habit